charts/stable/minecraft-java/values.yaml

```
@@ -233,7 +233,7 @@ workload:
             DEST_DIR: "{{.Values.persistence.backups.mountPath }}"
             SRC_DIR: "{{.Values.persistence.data.mountPath }}"
             SERVER_PORT: "{{ .Values.service.main.ports.main.port }}"
-            RCON_HOST: '{{ include "tc.v1.common.lib.chart.names.fullname" $ }}'
+            RCON_HOST: '{{ include "tc.v1.common.lib.chart.names.fullname" $ }}-rcon'
```

```suggestion
            RCON_HOST: '{{ printf "%s-rcon" (include "tc.v1.common.lib.chart.names.fullname" $) }}'
```
Personal preference, both work!

---

Much cleaner. Updated!